AddingTables_Project2.sql

```diff
@@ -0,0 +1,55 @@
+CREATE DATABASE SoniakB_MichaelD;
+
+CREATE TABLE Employee (
+	EmployeeID SMALLINT UNSIGNED,
+	Name varchar(60),
+	Salary decimal,
+	Bank_Acc_Number char(8),
+	NI_Number char(9),
+	Commission_Rate decimal,
+	Department SMALLINT UNSIGNED,
+	PRIMARY KEY (EmployeeID)
+	
+);
+
+ALTER TABLE Employee MODIFY EmployeeID int AUTO_INCREMENT;
```

Comment:
This could be inlined with the table creation. For example `EmployeeID SMALLINT UNSIGNED AUTO_INCREMENT`